documentation/modules/ROOT/examples/orgs_vars/superadmin/env/dev/controller_credentials.d/controller_credentials_gitlab_api_token.yml

```diff
@@ -5,7 +5,7 @@ controller_credentials:
     credential_type: "GitLab API Token"
     organization: "{{ orgs }}"
     inputs:
-      gitlab_token: "<CHANGE_ME>"
-      gitlab_email: "<CHANGE_ME>"
-      gitlab_user: "<CHANGE_ME>"
+      gitlab_token: "${GITLAB_USER}"
+      gitlab_email: "${GITLAB_EMAIL}" 
+      gitlab_user: "${GITLAB_TOKEN}"
```

Comment:
I'd prefer to use `<CHANGE_ME>` instead of leaving sensitive information into environment variables.

Reply:
the change is in order to ease up the workshop without manually editing too much content, but it's an open decision, so we can rollback these changes